infoschema/tables.go

```diff
@@ -278,14 +278,23 @@ var tableIDMap = map[string]int64{
 	TablePlacementPolicies:               autoid.InformationSchemaDBID + 79,
 }
 
+// columnInfo represents the basic column information of all kinds of INFORMATION_SCHEMA tables
 type columnInfo struct {
-	name      string
-	tp        byte
-	size      int
-	decimal   int
-	flag      uint
-	deflt     interface{}
-	comment   string
+	// represent column name
+	name string
+	// tp represent column type
```

Comment:
You could just say `tp is column type`, it is more concise.

Reply:
OK

Reply:
Done. please recheck. THKS